src/shared/models/crypto-asset-balance.model.ts

```diff
@@ -10,32 +10,30 @@ import { Money } from './money.model';
 
 export interface BitcoinCryptoCurrencyAssetBalance {
   readonly blockchain: 'bitcoin';
-  asset: BitcoinCryptoCurrencyAsset;
-  balance: Money;
   readonly type: 'crypto-currency';
+  readonly asset: BitcoinCryptoCurrencyAsset;
+  readonly balance: Money;
 }
 
 export interface StacksCryptoCurrencyAssetBalance {
   readonly blockchain: 'stacks';
-  asset: StacksCryptoCurrencyAsset;
-  balance: Money;
-  subBalance: Money;
   readonly type: 'crypto-currency';
+  readonly asset: StacksCryptoCurrencyAsset;
+  readonly balance: Money;
 }
 
 export interface StacksFungibleTokenAssetBalance {
   readonly blockchain: 'stacks';
-  asset: StacksFungibleTokenAsset;
-  balance: Money;
-  subBalance: Money;
   readonly type: 'fungible-token';
+  readonly asset: StacksFungibleTokenAsset;
+  readonly balance: Money;
```

Comment:
It's about mutability. Once created, we shouldn't never change the values.

<img width="465" alt="image" src="https://user-images.githubusercontent.com/1618764/202155088-8940849c-079a-41bf-9666-c12ff92a5688.png">

<img width="700" alt="image" src="https://user-images.githubusercontent.com/1618764/202155168-0e3fbabc-9d93-497c-9c5d-905d6bb89cd2.png">

> don't we need to change these when the metadata is queried

`addQueriedMetadataToInitializedStacksFungibleTokenAssetBalance` creates a new instance of `StacksFungibleTokenAssetBalance`, and doesn't mutate it directly, so doesn't trigger a type error.

Ideally, though, we should move away from creating blank/null data like:

https://github.com/hirosystems/stacks-wallet-web/blob/ab3a10a5c69def2b4c3c56c048bfdaaa9098cff5/src/app/query/stacks/balance/crypto-asset-balances.utils.ts#L43-L53

instead changing this fn to only work when it has all the data it needs, rather than adding it later. 
```
function createStacksFtCryptoAssetBalanceTypeWrapper(
  balance: BigNumber,
  metadata: CryptoAssetMetadata
)
```